src/js/models/v2/awards/additionalDetails/BaseContractAdditionalDetails.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * BaseContractAdditionalDetails.js
+ * Created by Lizzie Salita 3/5/18
+ */
+
+const BaseContractAdditionalDetails = {
+    populate(data) {
+        this.pricingCode = data.type_of_contract_pricing || '';
+        this.pricing = data.type_of_contract_pric_desc || '';
+
+        // Parent Award Details
+
+        this.idvType = data.idv_type || '--';
+        this.idcType = data.type_of_idc || '--';
+        this.idvAgencyId = data.referenced_idv_agency_iden || '--';
+        this.multipleIdv = data.multiple_or_single_award_i || '--';
```

Comment:
This looks like the old code points to `multiple_or_single_aw_desc` for this field.